sw/host/opentitanlib/src/test_utils/load_sram_program.rs

```diff
@@ -405,17 +412,31 @@ pub fn execute_sram_program(
     jtag: &mut dyn Jtag,
     prog_info: &SramProgramInfo,
     exec_mode: ExecutionMode,
+    skip_crc: bool,
 ) -> Result<ExecutionResult> {
     prepare_epmp(jtag)?;
     prepare_sram_ctrl(jtag)?;
+
     // To avoid unexpected behaviors, we always make sure that the return address
     // points to an invalid address.
     let ret_addr = 0xdeadbeefu32;
     log::info!("set RA to {:x}", ret_addr);
     jtag.write_riscv_reg(&RiscvReg::Gpr(RiscvGpr::RA), ret_addr)?;
-    // The SRAM program loader expects the CRC32 value in a0
-    log::info!("set A0 to {:x} (crc32)", prog_info.crc32);
-    jtag.write_riscv_reg(&RiscvReg::Gpr(RiscvGpr::A0), prog_info.crc32)?;
+
+    // Potentially skip CRC check.
+    if skip_crc {
+        // The SRAM program loader will skip the CRC32 check if a0 is a magic value.
+        log::info!(
+            "skip CRC by setting A0 to {:x} (crc32)",
+            SRAM_MAGIC_SP_CRC_SKIPPED
+        );
+        jtag.write_riscv_reg(&RiscvReg::Gpr(RiscvGpr::A0), SRAM_MAGIC_SP_CRC_SKIPPED)?;
```

Comment:
@benjbender if you want to skip the CRC check just set A0 to the magic value of `0xbeefcafe` when you load the SRAM program over JTAG.

If we want to turn the CRC checks on later, it will be easy to do so too